packages/react-native-win32/flow-typed/npm/pretty-format_v26.x.x.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * (c) Facebook, Inc. and its affiliates. Confidential and proprietary.
+ *
+ * @flow strict
+ * @format
+ */
+
```

Comment:
This is synchronized with the RN Core OSS source tree. Guessing someone forgot to update the copyright headers when bringing this into OSS? FYI @TheSavior

Reply:
Hmm, yeah, not sure why our license linter didn't force us to swap it. Thanks for flagging